doc/source/python_reference.rst

```diff
@@ -579,6 +579,8 @@ Functions:
    apply_inverse_epochs
    apply_inverse_raw
    compute_rank_inverse
+   psd_epochs_source
+   psd_raw_source
```

Comment:
@dengemann you mean you like and want to keep the prefixes `compute_` and `plot_`, right? That to me seems consistent. What seems inconsistent is changing it to be `psd_epochs_source` instead of e.g. `compute_source_epochs_psd`. Is that what you also mean?